cloud-run-single/1-apps/terraform.tfvars.sample

```diff
@@ -3,3 +3,28 @@ lbs_config = {
     domain = "your-domain.com"
   }
 }
+
+## Uncomment config below for GPU Workloads
+# cloud_run_configs = {
```

Comment:
Should we put this in another terraform.tfvars.sample to make it more clear? like terraform.tfvars.sample-gemma?

Also we can insert in the tfvars only the values that override the default. For example, we can exclude ingress, max_instance_count, service_invokers, .....